Description: Upgrade of Jenkins codebase to use ASM3 rather than ASM2.
 This is required because Ubuntu/Debian builds against later versions
 of libraries which have already transitioned.
Author: James Page <james.page@ubuntu.com>
Forwarded: no

Index: jenkins/core/src/main/java/hudson/util/SubClassGenerator.java
===================================================================
--- jenkins.orig/core/src/main/java/hudson/util/SubClassGenerator.java	2012-02-28 16:49:30.829862282 +0000
+++ jenkins/core/src/main/java/hudson/util/SubClassGenerator.java	2012-04-26 14:15:34.236629652 +0100
@@ -44,7 +44,7 @@
     }
 
     public <T> Class<? extends T> generate(Class<T> base, String name) {
-        ClassWriter cw = new ClassWriter(false,false);//?
+        ClassWriter cw = new ClassWriter(0);//?
         cw.visit(49, ACC_PUBLIC, name.replace('.', '/'), null,
                 Type.getInternalName(base),null);
 
